infoportal-server/src/core/EmailService.ts

```diff
@@ -0,0 +1,34 @@
+import {logger} from '../helper/Logger'
+import {GlobalEvent} from './GlobalEvent'
+import sendEmail from './EmailHelper'
```

Comment:
Use DI for this too. It's cleaner to have a class `EmailHelper`, with a method `send` that you instanciate in the constructor.